examples/cdi-webapp/pom.xml

```diff
@@ -182,6 +199,18 @@
                     <classpathScope>test</classpathScope>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <forkMode>always</forkMode>
+                    <enableAssertions>false</enableAssertions>
+                    <!-- TODO remove after jakartification -->
+                    <excludes>
+                        <exclude>org/glassfish/jersey/examples/cdi/resources/EchoParamBeanTest.java</exclude>
```

Comment:
Not sure I follow. You have excluded all tests in `jakartification_exclude_tests` profile and here you exclude one more test extra....But ok.

Reply:
I would keep this the one removal refers to jakartification w/o Weld jakartified, and all tests are skipped due to jakartified weld